cpp/src/IceGrid/Internal.ice

```diff
@@ -279,15 +279,15 @@ interface NodeSession
     ["cpp:const"] idempotent NodeObserver* getObserver();
 
     /// Ask the registry to load the servers on the node.
-    ["amd", "cpp:const"] idempotent void loadServers();
+    ["amd"] ["cpp:const"] idempotent void loadServers();
```

Comment:
I assume you kept some instances (tests) for the "," syntax?

Reply:
Yes, the `escape` tests all still use the old syntax, and so do the metadata-specific tests in `cpp/test/Slice`.
Everything else was updated.